docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -28,8 +37,8 @@ appropriate for your setup:
 Multiple ClickHouse servers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you have multiple ClickHouse instances you need to configure one catalog for
-each instance. To add another catalog:
+If you have a cluster with multiple ClickHouse servers you need to configure one
```

Comment:
This suggests that we support the "cluster" deployment of ClickHouse - which we don't know if it works. I think the earlier wording simply meant that if you have multiple ClickHouse servers each one needs their own catalog files.

Reply:
Got it. Fixed.